service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseLinkNotificationHandler.java

```diff
@@ -22,34 +22,34 @@
 
 @Component
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
-public class CaseManagementOrderCaseLinkNotificationHandler {
+public class CaseLinkNotificationHandler {
```

Comment:
Ahh, I didn't mean the class names in my previous comments, just variables and methods

Reply:
Also these changes probably apply to the other classes as well